Resources/contribute/template/docs-template.md

```diff
@@ -1,58 +0,0 @@
----
-layout: docs
-title: <title>
```

Comment:
Why are you deleting this file ? We also have samples on web-sites apart from docs. Please re-write this or leave it be.

Reply:
We have samples template for the same. Deleting this to ensure that people do not commit docs to WOD.